spec/support/fake_ansible_repo.rb

```diff
@@ -106,14 +55,11 @@
 #     var_3: baz
 #
 
-require "rugged"
-
-require "pathname"
-require "fileutils"
+require_relative "fake_git_repo"
 
 module Spec
   module Support
-    class FakeAnsibleRepo
+    class FakeAnsibleRepo < Spec::Support::FakeGitRepo
```

Comment:
Super minor, but this shouldn't need the qualifier on the superclass since it's in the same namespace.  Not holding up merge for this.

```suggestion
    class FakeAnsibleRepo < FakeGitRepo
```